test/Microsoft.ML.Tests/Transformers/DateTimeTransformerTests.cs

```diff
@@ -133,7 +133,50 @@ public void CanUseDateFromColumn()
             Done();
         }
 
-        [NotCentOS7FactAttribute]
+        [NotCentOS7Fact]
+        public void CanUseDateFromColumnDateTimeType()
+        {
+            // Future Date - 2025 June 30
```

Comment:
If this code survives this date...will it still be the future? ;-)

Reply:
Good point. I'll remove the "Future" comment.